x-pack/plugins/security/public/account_management/user_profile/user_profile.test.tsx

```diff
@@ -292,7 +292,7 @@ describe('useUserProfileForm', () => {
         </Providers>
       );
 
-      expect(testWrapper.exists('EuiButtonGroup[data-test-subj="themeMenu"]')).toBeFalsy();
+      expect(testWrapper.exists('[data-test-subj="darkModeButton"]')).toBeFalsy();
```

Comment:
There is no longer a component with this data-test-subject, so this will always be falsy. It should check `EuiKeyPadMenu[data-test-subj="themeMenu"]`.
```suggestion
      expect(testWrapper.exists('EuiKeyPadMenu[data-test-subj="themeMenu"]')).toBeFalsy();
```

Reply:
Thank you for the suggestion! This works locally for me too so I'll update this branch.